apps/wallet-connect/src/hooks/useWalletConnectV2.tsx

```diff
@@ -217,7 +218,7 @@ const useWalletConnectV2 = (
                 accounts: safeAccount || [
                   `${EVMBasedNamespaces}:${safe.chainId}:${safe.safeAddress}`,
                 ],
-                methods: EIP155Namespace.methods,
+                methods: [...EIP155Namespace.methods, ...(optionalEIP155Namespace?.methods || [])],
```

Comment:
If you use `concat`, you need not use a fallback (and it's also more performant).
```suggestion
                methods: EIP155Namespace.methods.concat(optionalEIP155Namespace?.methods),
```